recipes/morphotactics/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "morphotactics" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dmort27/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0c0ff9fc35acb9281438e4412f40c46f515cb1f0998a0ab301a0d1ce0c2a07bc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win or py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cython >=0.29
+    - openfst ==1.8.1
```

Comment:
Cython doesn't seem to be used anywhere, and `openfst` doesn't seem to be needed at package build time.
```suggestion
```